### DISCUSSION TITLE
Does extreme quality use photometric?

### DISCUSSION THREAD

**Discussion by SangwooRyu** [colmap/colmap#2890] [Q&A] [answered]
I'm looking on quality option in automatic reconstruction, and I found below line.

https://github.com/colmap/colmap/blob/3254263266949413d7c669e44abeb4a7c2670a8b/src/colmap/controllers/automatic_reconstruction.cc#L321

It seems, the input type of stereo fusion is 'geometric' when Quality::HIGH, and 'photometric' when else.

But why not using 'geometric' also for Quality::EXTREME?

**Comment by ahojnnes** [comment, marked as answer]
Thanks for reporting the issue. This should be fixed in https://github.com/colmap/colmap/pull/2893
